### DISCUSSION TITLE
Activate another template from external

### DISCUSSION THREAD

**Discussion by knallt** [sidoh/epaper_templates#64] [Q&A]
Hey @sidoh 

This is really amazing work. Thank you! 
This has worked for me everything on the first try!

I have a question: Is it possible, e.g. via API, to activate another template or to switch between templates?
Thank you!

**Comment by sidoh** [comment]
Hey @knallt,

Yeah, definitely possible. The UI interacts exclusively with a REST API.

The active template is stored in the settings document, which you can patch with a request like this:

```
curl -X PUT -H 'content-type: application/json' -d '{"display.template_name":"/t/office_dashboard_covid2.json"}' http://epaper-display.sidoh.org/api/v1/settings
```

**Comment by knallt** [comment]
top, works. Thx!
